src/main/java/racing/domain/TryNumber.java

```diff
@@ -4,28 +4,32 @@
 
     private static final int MIN_TRY_NUMBER = 1;
 
-    private final int tryNumber;
+    private int tryNumber;
 
-    public TryNumber(String tryNumber) {
-        this.tryNumber = toInt(tryNumber);
-        isValidTryNumber(this.tryNumber);
+    public TryNumber(int tryNumber) {
+        checkValidTryNumber(tryNumber);
+        this.tryNumber = tryNumber;
     }
 
-    private int toInt(String tryNumber) {
-        try {
-            return Integer.parseInt(tryNumber);
-        } catch (NumberFormatException nfe) {
-            throw new NumberFormatException("[ERROR] 숫자만 입력 해주세요.");
+    private void checkValidTryNumber(final int tryNumber) {
+        if (tryNumber < MIN_TRY_NUMBER) {
+            throw new IllegalArgumentException("[ERROR] 시도 횟수는 최소 1회 이상이어야 합니다.");
         }
     }
 
-    private static void isValidTryNumber(final int tryNumber) {
-        if (tryNumber < MIN_TRY_NUMBER) {
-            throw new IllegalArgumentException("[ERROR] 시도 횟수는 최소 1회 이상이어야 합니다.");
+    public boolean hasNextStep() {
+        if (this.tryNumber == 0) {
+            return false;
         }
+        return true;
+    }
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요! 😃
```suggestion
    public boolean hasNextStep() {
        return this.tryNumber != 0;
```

또한 추가적으로 `0` 또한 매직넘버라고 볼 수 있을 것 같아요. 🤔

Reply:
오! 한 문장으로 줄일 수 있는 메서드는 이렇게 가독성 좋은 방법으로 적용해보겠습니다! 감사합니다 😄